quickstarts/mlops/truera/config.yml

```diff
@@ -0,0 +1,50 @@
+# Name of the quickstart (required)
+name: truera
+
+# Displayed in the UI
+title: Truera
+
+# Long-form description of the quickstart (required)
+description: |
+  TruEra provides best in class AI Quality diagnostics and monitoring solution. With this integration, TruEra and NewRelic users can go beyond standard accuracy, input and output drift monitoring by identifying consequential data drift, and other AI Quality diagnostics such as fairness, conceptual soundness and segment behavior. When combined with NewRelic’s comprehensive observability capabilities, MLOps teams can correlate signals throughout the ML lifecycle.
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
```

Comment:
I'd recommend a more simplified summary as this is used in the preview tile on the main I/O catalog page.

Reply:
You can try:
`Go beyond standard accuracy, input and output drift monitoring and identify consequential data drift and other AI quality diagnostics as well as correlate signals throughout the machine learning lifecycle.`